app/Models/LdapAdConfiguration.php

```diff
@@ -0,0 +1,245 @@
+<?php
```

Comment:
This seems pretty thin, and just takes existing Snipe-IT configuration variables and jams then into the Ldap model object. Is there a way to just take those values directly, obviating this?

Reply:
The configuration does a couple of things. It sets the configuration for the ldap connection by processing the Snipe-IT settings. Some of those settings are converted for easier use, I also plan to cache the settings as they not updated frequently.